src/Components/Form/Form.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  state = {
+    errorMessage: '',
+  };
+
+  setErrorMessage = data =>
+    this.setState({
+      errorMessage: data,
+    });
+
+  submitForm = event => {
+    event.preventDefault();
+
+    const {
+      target: {
+        location: { value },
+      },
+    } = event;
+    const API_ENDPOINT = `${process.env.REACT_APP_WEATHER_API_URL}${value}${process.env.REACT_APP_WEATHER_API_KEY}`;
+    const { setWheaterData } = this.props;
+
+    fetch(API_ENDPOINT)
+      .then(response => response.json())
+      .then(json => setWheaterData(json))
+      .catch(error =>
+        this.setErrorMessage(
+          'There was an error while fetching data. Please refresh the page to try again.'
+        )
+      );
+  };
+
+  render() {
+    const { errorMessage } = this.state;
+
+    if (errorMessage) {
+      console.error(errorMessage);
```

Comment:
User will not go to console to see error. He is not a programmer. 

You can create an ErrorMessage.jsx component which will take 
`
const ErrorMessage = ({children}) => (<div className="erorr-message"> <p> {children} </p> </div>`

Add some red styles to it 😄   and use in render function with ternary operator

```
render () {
 return (

    errorMessage ? ( <ErrorMessage>  {errorMessage} </ErrorMessage> : ..... rest of the HTML when no error appears.
  )
}
```